### DISCUSSION TITLE
Learning from gradients of a multivariate function

### DISCUSSION THREAD

**Discussion by albapa** [cornellius-gp/gpytorch#1845] [Q&A]
Is it somehow possible to learn a multivariate function f(x,y) from its partial derivatives (and function observations)? The example only shows this for a f(x) univariate function. I'd be grateful for any suggestions or pointers.

**Comment by gpleiss** [comment]
Does this tutorial help? https://docs.gpytorch.ai/en/stable/examples/08_Advanced_Usage/Simple_GP_Regression_Derivative_Information_2d.html
